test/powershell/Modules/Microsoft.PowerShell.Management/Set-Location.Tests.ps1

```diff
@@ -19,64 +26,33 @@ Describe "Set-Location" -Tags "CI" {
         { Set-Location env: }   | Should Not Throw
     }
 
-    It "Should be able use the cd alias without error" {
-        { cd $target } | Should Not Throw
-    }
-
-    It "Should be able to use the chdir alias without error" {
-        { chdir $target } | Should Not Throw
-    }
-
-    It "Should be able to use the sl alias without error" {
-        { sl $target } | Should Not Throw
-    }
-
     It "Should have the correct current location when using the set-location cmdlet" {
         Set-Location $startDirectory
 
         $(Get-Location).Path | Should Be $startDirectory.Path
     }
 
-    It "Should have the correct current location when using the cd alias" {
-        cd $target
-
-        $(Get-Location).Path | Should Be $target
-    }
-
-    It "Should have the correct current location when using the chdir alias" {
-        chdir $target
-
-        $(Get-Location).Path | Should Be $target
-    }
-
-    It "Should have the correct current location when using the chdir alias" {
-        sl $target
-
-        $(Get-Location).Path | Should Be $target
-    }
```

Comment:
if any of those aliases are removed, these tests can catch it. So maybe we should keep these 3 tests?